src/dirtyfields/dirtyfields.py

```diff
@@ -64,9 +64,12 @@ def _as_dict(self, check_relationship, include_primary_key=True):
                 # psycopg2 returns uncopyable type buffer for bytea
                 field_value = str(field_value)
 
+            # Use the column name (instead of the relationship name) if it's a
+            # foreign key.
+            key = field.attname if hasattr(field, 'attname') else field.name
```

Comment:
I could be wrong, but I assume it's possible in older versions of django. I couldn't find any solid documentation on attname, but I used the example for `MyModel._meta.get_all_field_names` from here: https://docs.djangoproject.com/en/1.10/ref/models/meta/#migrating-from-the-old-api